### PR TITLE
Apply teal background to UI shell header

### PR DESCRIPTION
**Pull request by denniskigen**
Applies the OpenMRS branding teal color (#005d5d) as the UI shell header background color per the [new designs](https://app.zeplin.io/project/5f7223cfda10f94d67cec6d0/screen/5fa4407a5f96c602cc7114c6).

Screenshot from 2021-02-02 23-14-18

### CODE REVIEW

File: src/components/navbar.component.tsx [hunk 1]
```diff
@@ -55,7 +56,7 @@ const Navbar: React.FC<NavbarProps> = ({ user, onLogout, allowedLocales }) => {
   }, []);
 
   return (
-    <div ref={headerRef}>
+    <div ref={headerRef} className={styles.navbar}>
```
Comment (jonathandick) [inline src/components/navbar.component.tsx:60]: Thanks for this @denniskigen. 

Do you have any thoughts on how to make this configurable?

**Review by FlorianRappl — COMMENTED** [review]
Great - can you also change to use the logo as the designs suggest?

**Comment by denniskigen** [comment]
Yeah, tomorrow.

File: src/components/navbar.component.tsx (hunk 1 above)
Comment (brandones) [reply src/components/navbar.component.tsx:60]: My idea is to make it configurable via global overrides of CSS variables. Maybe esm-styleguide could house that piece of configuration. So you'd provide like
```jsx
{
  "@openmrs/esm-styleguide": {
    "colors": {
      "brand-01": "#0000FF"
    }
  }
}
```

and esm-styleguide would override the `brand-01` CSS var with the provided hex value.

Comment (jonathandick) [reply src/components/navbar.component.tsx:60]: so we would be importing "brand-01" into this repo? Could you provide code to make crystal clear?

Comment (brandones) [reply src/components/navbar.component.tsx:60]: Nothing in this repo would need to change.

Somewhere in esm-styleguide:

```jsx
const config = getConfig("@openmrs/esm-styleguide");
for (let [varName, value] of Object.entries(config.colors)) {
  document.documentElement.style.setProperty('--' + varName, value);
}
```

Screenshot from 2021-02-02 20-56-07

Comment (FlorianRappl) [reply src/components/navbar.component.tsx:60]: Why do we need any JS for this?

Just allow specifying some CSS file - overriding CSS variables / CSS in general should be done by CSS imho.

Comment (brandones) [reply src/components/navbar.component.tsx:60]: Sure, that's also reasonable. Using config, the implementer would choose colors in the implementer-tools config panel and they would be part of the config file. Using CSS, the implementer would write a CSS file, and we would need some way of making it easy for an implementer to provide that CSS file, perhaps by adapting the `assemble` tooling. I think either is fine from an implementer-UX perspective.

Comment (denniskigen) [reply src/components/navbar.component.tsx:60]: Update:Brand logo added (courtesy Ciaran)

Screenshot from 2021-02-03 20-58-36

Comment (FlorianRappl) [reply src/components/navbar.component.tsx:60]: Yes that's one way @brandones - the other way is that the app shell has a config like `cssUrls` where you can add an array of valid CSS stylesheet URLs. The URLs from that array could be added to the application via `link` elements added to the `head`.

**Review by FlorianRappl — APPROVED** [review]
LGTM - thanks for bringing in the logo!

**Review by FlorianRappl — APPROVED** [review]
LGTM!

**Pull request merged**